src/JavaLang.jl

```diff
@@ -0,0 +1,59 @@
+# Module to represent the base java package: java.lang
+# This modules has basic definitions that should be exported by default
+module JavaLang
+
+export JString, JObject, new_string, equals
+
+using JavaCall.JNI
+using JavaCall.Core
+using JavaCall.CodeGeneration
+using JavaCall.Conversions
+
+# Replace by dynamic generation of JObject. Just to show that it is possible.
+eval(generateblock(
+    generatetype(:JObject),
+    generatemethod(:convert, [:(::Type{jobject}), :(x::JObject)], :(x.ref)),
```

Comment:
JObject is an abstract type and thus doesn't have a `ref` field

Reply:
Yes, but the structure that will be implementing it will have, so that it can store the reference to the object. We need to define the conversions for all types. However, I think this will later be replaced by code generation. Also, when we generate the code we will also generate a JObjectImpl.